verticaltabs.js

```diff
@@ -457,6 +471,25 @@ VerticalTabs.prototype = {
 
     let enterTimeout = -1;
 
+    let exit = (event) => {
+      this.mouseExited();
+      if (enterTimeout > 0) {
+        window.clearTimeout(enterTimeout);
+        enterTimeout = -1;
+      }
+      if (mainWindow.getAttribute('tabspinned') !== 'true') {
+        if (!leftbox.contextMenuOpen){
+          leftbox.removeAttribute('expanded');
+        }
+
+        this.clearFind();
+        let tabsPopup = document.getElementById('alltabs-popup');
+        if (tabsPopup.state === 'open') {
+          tabsPopup.hidePopup();
+        }
+      }
+    };
+
     let enter = (event) => {
       this.mouseEntered();
       if (event.type === 'mouseenter' && leftbox.getAttribute('expanded') !== 'true') {
```

Comment:
Oh!  But the `if (event.pageX <= 4)` below is why we were calling this on `mousemove`, because if we throw the cursor all the way to the left, we won't (necessarily) get a `mouseenter` event all the way over there, so we'll need to check for this in both `mouseenter` and `mousemove` events.